README.md

```diff
@@ -43,7 +43,7 @@ To deploy the application:
    application](https://developers.google.com/appengine/docs/python/tools/uploadinganapp) with
 
    ```
-   appcfg.py -A <your-project-id> --oauth2 update .
+   appcfg.py -A <your-project-id> -V v1 update .
```

Comment:
It looks like it's a separate call:

    appcfg.py set_default_version -V v1 -A <your-project-id>